subprojects/core/src/main/java/org/gradle/api/internal/component/UsageContext.java

```diff
@@ -22,6 +22,7 @@
 import org.gradle.api.artifacts.PublishArtifact;
 import org.gradle.api.attributes.HasAttributes;
 import org.gradle.api.attributes.Usage;
+import org.gradle.api.capabilities.CapabilityDescriptor;
```

Comment:
The more I see this, the more I think maybe it should just be named `Capability`

Reply:
will do